recipes/scanpy/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "scanpy" %}
+{% set version = "1.7.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ad5e7afdf1948753cd27f6f27ee904df1ce455855eec8520985effe86c53c630
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - "scanpy = scanpy.cli:console_main"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - flit-core
+    - setuptools_scm
+    - pytoml
+  run:
+    - python >=3.6
+    - anndata >=0.7.4
+    - numpy >=1.17
+    - matplotlib >=3.1.2
```

Comment:
I think it’s recommended to depend on `matplotlib-base` if you don’t require the Qt frontend and only need the core parts of the library. See https://github.com/conda-forge/conda-forge.github.io/issues/933

Reply:
OK, thank you!